controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -324,15 +324,16 @@ private Optional<Version> oldestInstalledPlatform(JobId job) {
     /** Returns the oldest Vespa version installed on any active or reserved production node for the given application. */
     public Optional<Version> oldestInstalledPlatform(Application application) {
         return controller.jobController().deploymentStatus(application).jobs()
-                         .production().asList()
-                         .stream()
+                         .production()
```

Comment:
Doesn't `production()` imply `not().test()`?

Reply:
Production tests are production tests Ü